circle.yml

```diff
@@ -29,6 +29,8 @@ dependencies:
     - bundle install
     - gem update fastlane spaceship
     - bower install
+    - ember build
+    - curl https://api.rollbar.com/api/1/sourcemap/download -F access_token=b53733a143b0422c8b1e5bb8624e8683 -F version="0" -F minified_url=https://app-staging.goodcity.hk/assets/goodcity.js
```

Comment:
Please remove the access token from circle.yml and use an ENV var in Circle instead.
You will need to cycle the token now so it is not able to be used again.

Reply:
@steveyken , I missed that. Thanks for pointing it out 👍